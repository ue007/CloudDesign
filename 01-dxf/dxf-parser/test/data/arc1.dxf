  0
SECTION
  2
ENTITIES
  0
ARC
  5
B484
330
3706
100
AcDbEntity
  8
FG-Dtl-Dim
  6
ByBlock
 62
     0
370
    -2
100
AcDbCircle
 10
650.0541823996996
 20
347.5909732684974
 30
0.0
 40
0.7604262403327394
100
AcDbArc
 50
338.8998819521181
 51
355.0752244008874
0
ENDSEC
  0
EOF