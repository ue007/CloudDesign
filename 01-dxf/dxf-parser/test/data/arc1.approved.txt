{
  "entities": [
    {
      "type": "ARC",
      "handle": "B484",
      "ownerHandle": "3706",
      "layer": "FG-Dtl-Dim",
      "lineType": "ByBlock",
      "colorIndex": 0,
      "color": 0,
      "lineweight": -2,
      "center": {
        "x": 650.0541823996996,
        "y": 347.5909732684974,
        "z": 0
      },
      "radius": 0.7604262403327394,
      "startAngle": 5.914918774684569,
      "endAngle": 6.197231758053196,
      "angleLength": 0.28231298336862665
    }
  ]
}
